client/components/users/UserQuizHistoryTaken.jsx

```diff
@@ -65,88 +98,56 @@ function UserQuizHistoryTaken(userId) {
-              variant="outlined"
-              color="primary"
-              onClick={ handleRedirect }
-              name={ quiz.id_quiz }
-            >
-              Retake Quiz
-            </button>
-          </Grid>
-        ))
-        : null
-      }
-      <Button
-        variant="contained"
-        variant="outlined"
-        color="primary"
-        onClick={ handleClose }
-      >
-        Close
-      </Button>
-    </Grid>
-  )
-
   return (
-    <Grid item xs>
-      <Card style={{ height: "100%" }}>
-        <CardHeader
-          title="Quizzes Taken"
-          titleTypographyProps={{ variant: "body2" }}
-          style={{ padding: 10 }}
-        />
-        <CardContent>
-          <Grid>
-            {quizList.length
-              ? quizList.slice(0, 5).map(quiz => (
-                <Grid item>
-                  <h4>{ quiz.name }</h4>
-                  <h4>Score: { quiz.correct_answer_count }/{ quiz.correct_answer_count + quiz.incorrect_answer_count} ({(Number(quiz.correct_answer_count)/Number(quiz.correct_answer_count + quiz.incorrect_answer_count) * 100).toFixed(0) }%)</h4>
-                  <button
-                    variant="contained"
-                    variant="outlined"
-                    color="primary"
-                    onClick={ handleRedirect }
-                    name={ quiz.id_quiz }
-                  >
-                    Retake Quiz
-                  </button>
-                </Grid>
-              ))
-              : null
-            }
-            {count > 5
-              ? <Grid item>
-                <Button
-                  variant="contained"
-                  variant="outlined"
-                  color="primary"
-                  onClick={ handleOpen }
+    <TableContainer>
+      <Table className={ classes.table } stickyHeader={ true } aria-label='quizzes'>
+        <TableHead>
+          <TableRow>
+            <TableCell align="center" style={{ fontSize: 22 }}>Name</TableCell>
+            <TableCell align="center" style={{ fontSize: 22 }}>Category</TableCell>
+            <TableCell align="center" style={{ fontSize: 22 }}>Score</TableCell>
+            <TableCell align="center" style={{ fontSize: 22 }}>Date Taken</TableCell>
+            <TableCell align="center" style={{ fontSize: 22 }}>Actions</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {quizList.length
+            ? quizList.map(quiz => (
+              <TableRow key={ quiz.id }>
+                <TableCell align="center" style={{ fontSize: 16 }}>{ quiz.name }</TableCell>
+                <TableCell align="center" style={{ fontSize: 16 }}>{ quiz.category }</TableCell>
+                <TableCell align="center" style={{ fontSize: 16 }}>{ quiz.correct_answer_count }/{ quiz.correct_answer_count + quiz.incorrect_answer_count } ({ (Number(quiz.correct_answer_count)/Number(quiz.correct_answer_count + quiz.incorrect_answer_count) * 100).toFixed(0) }%)</TableCell>
```

Comment:
If you can, try breaking this line into multiple lines.

Reply:
....but this is how you have it in your code too.....lol. regardless, made the changes you requested!

Reply:
Lmao stay tuned for my PR!